### DISCUSSION TITLE
机器人mode 栏显示for的问题

### DISCUSSION THREAD

**Discussion by yamiiro** [Tsuk1ko/CQ-picfinder-robot#226] [问题求助] [answered]
0|cqps     | - /root/cq-picsearcher-bot/src/plugin/akhr/index.js
0|cqps     | - /root/cq-picsearcher-bot/main.js
0|cqps     | - /root/cq-picsearcher-bot/index.js
0|cqps     |     at loadCanvasModule (/root/cq-picsearcher-bot/src/plugin/akhr/akhr.draw.js:10:14)
0|cqps     |     at CqpsEventEmitter.onceInit (/root/cq-picsearcher-bot/src/event.js:16:22)
0|cqps     |     at Object.<anonymous> (/root/cq-picsearcher-bot/src/plugin/akhr/akhr.draw.js:15:7)
0|cqps     |     at Generator.next (<anonymous>) {
0|cqps     |   code: 'MODULE_NOT_FOUND',
0|cqps     |   requireStack: [
0|cqps     |     '/root/cq-picsearcher-bot/src/plugin/akhr/akhr.draw.js',
0|cqps     |     '/root/cq-picsearcher-bot/src/plugin/akhr/index.js',
0|cqps     |     '/root/cq-picsearcher-bot/main.js',
0|cqps     |     '/root/cq-picsearcher-bot/index.js'
0|cqps     |   ]
0|cqps     | }
以上是日志，go-cqhttp最新的.

**Comment by yamiiro** [comment]
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! cq-picsearcher-bot@2.29.12 test: `node index.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the cq-picsearcher-bot@2.29.12 test script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/cq-picsearcher-bot/node_cache/_logs/2021-07-14T10_59_23_812Z-debug.log

**Comment by Tsuk1ko** [comment, marked as answer]
请看 wiki，按找不到模块处理

**Reply by yamiiro** [reply]
好的，我试试
